fms/models/llama.py

```diff
@@ -421,7 +454,22 @@ def factory(**kwargs):
 models.register_model(_architecture_name, "13b", _llama_factory_factory(_13b_config))
 models.register_model(_architecture_name, "70b", _llama_factory_factory(_70b_config))
 models.register_model(
-    _architecture_name, "llama3.8b", _llama_factory_factory((_8b_llama3_config))
+    _architecture_name, "3.8b", _llama_factory_factory((_8b_llama3_config))
```

Comment:
I'm still concerned about this naming convention:

https://github.com/foundation-model-stack/foundation-model-stack/pull/266#issuecomment-2245751136

we have "llama3.8b" when "llama 3.1" was released today. we could end up with llama3.1.8b, llama3.1.70b, etc. I think we need some better way to differentiate the model version and the model size, like an underscore or dash.

Reply:
let me know what you think of the new way I'm doing it